app/fullService/api/setPassword.ts

```diff
@@ -0,0 +1,28 @@
+import axiosFullService from '../axiosFullService';
+
+const SET_PASSWORD_METHOD = 'set_password';
+
+type SetPasswordParams = {
+  password: string;
+};
+
+type SetPasswordResult = {
+  success: boolean;
+};
+
+const setPassword = async (
+  { password }: SetPasswordParams,
+): Promise<SetPasswordResult> => {
+  const { result, error } = await axiosFullService(SET_PASSWORD_METHOD, {
```

Comment:
Shouldn't this `await` be in a `try...catch` statement?

Reply:
Probably! I plan on going back and cleaning up the axios wrapper.

I'm thinking it looks like this:
axios wrapper (axiosFullService) --> api calls (e.g., `setPassword`) --> service functions (a service chains necessary api calls to solve a singular purpose) --> fullServiceContext actions (the api exposed to the react guis that trigger an update to state)

I wouldn't look too hard at this WIP. Just yet! (it will become priority in a few days)